### DISCUSSION TITLE
Welcome to EasyButton Discussions!

### DISCUSSION THREAD

**Discussion by evert-arias** [evert-arias/EasyButton#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pnaybour** [comment]
I want to be able the change the number of presses and delay time for a sequence of presses after a callback has been set up. Shall I start a new branch or can it already be fine in some way?

**Comment by elC0mpa** [comment]
Hello, I am José Gabriel Companioni, and I have been an active contributor of this library, that is why @evert-arias made me a colaborator, please feel free to share your ideas with us

**Reply by pnaybour** [reply]
Jose @elC0mpa great to meet you. Nice work in the library. 

I found class Sequence in the software. I think it will enable me to change the number of presses in a sequence during run time in the loop(). It's not documented, but I will test it later. I have two button so I am not sure how to use this class for each button. And advice would be great full received.

**Comment by mdleisure** [comment]
Hello, 
A while ago I made a request to use easybutton with expansion boards. Unfortunately with lockdown etc I never got to implement this into my design when it was added and have recently picked it back up. 
Would it be possible to see an example of the code to use with an expansion board. 
Personally I am using esp8266 and mcu 2317 for testing but any example I could use to learn from would be a help. 
Thanks

**Comment by damarcrazy** [comment]
Hello my name is Dario Amaris, I greet you from Colombia
Thanks for the work you've done with the EasyButton library.
I hope it is not too much to ask if it is possible to help me with something that I do not know how to do.

I want in a part of the code to place a while loop that does nothing until
a button is pressed. I know how to do it without the library while (digitalRead(button) ==1 ){ }
but with her I don't know. please give me an idea

Thank you
